blackduck-security-task/src/blackduck-security-task/tools-parameter.ts

```diff
@@ -218,7 +218,9 @@ export class BridgeCliToolsParameter {
 
         if (inputs.POLARIS_PR_COMMENT_SEVERITIES) {
           polData.data.polaris.prcomment.severities =
-            inputs.POLARIS_PR_COMMENT_SEVERITIES.filter((severity) => severity);
```

Comment:
Please apply similar fix for other parameters as well where we accept comma separated string

Reply:
As I have checked, it is already applied for other comma separated inputs.